src/components/calcite-tip-group/calcite-tip-group.scss

```diff
@@ -5,6 +5,4 @@
 }
 
 ::slotted(calcite-tip) {
-  border: none;
-  max-width: var(--calcite-tip-max-width);
-}
+  border: none;
```

Comment:
did you delete this ending curly bracket or is github messing with me?

Reply:
not sure how this made it into this PR. reverting this back as this is in another branch/PR